/daheng_cam_driver:
  ros__parameters:
    camera_type: 0
    frame_id: daheng_cam
    debug: true
    save_video: false
    using_port: false
    save_path: "/recorder/video/"
    config_path: "/config/daheng_cam_param.ini" #加载参数配置文件路径

/mvs_cam_driver: # 配置文件在camera_driver包的config目录下
  ros__parameters:
    camera_type: 2
    frame_id: mvs_cam
    using_port: false
    save_video: false
    save_path: "/recorder/video/"
    config_path: "/config/mvs_cam_param.config" #加载参数配置文件路径
    
/usb_cam_driver:
  ros__parameters:
    cam_id: 0
    camera_type: 3
    frame_id: usb_cam
    image_width: 990
    image_height: 720
    fps: 60
    debug: true
    using_video: true
    using_ros2bag: false
    video_path: "/recorder/video/new_buff.mp4"
    save_video: false
    save_path: "/recorder/video/"
    using_port: false
    
/hik_cam_driver:
  ros__parameters:
    cam_id: 0
    camera_type: 1
    frame_id: hik_cam
    image_width: 1440
    image_height: 1080
    exposure_time: 4000
    exposure_gain: 7
    exposure_gain_b: 0
    exposure_gain_g: 0
    exposure_gain_r: 0
    balance_b: 1690
    balance_g: 1024
    balance_r: 2022
    debug: true
    save_video: false
    save_path: "/recorder/video/"
    using_port: false

/serialport:
  ros__parameters:
    debug_without_com: true
    baud: 115200
    using_port: false
